student.rb

```diff
@@ -23,8 +23,7 @@ def to_h
       age: @age,
       name: @name,
       parent_permission: @parent_permission,
-      rentals: @rentals,
-      classroom: @classroom
+      rentals: @rentals
     }
   end
 end
```

Comment:
- [ ] Kindly write unit tests for all methods (including initialize method if need be) for the following classes in the application:
- `Decorator` class
- `CapitalizeDecorator` class
- `TrimmerDecorator` class

Remember it is a requirement to test all classes. Kindly test these classes listed above 😎 💻